### PR TITLE
Optimise two

### PR DESCRIPTION
**Pull request by pbiology**
This PR is a small update with various improvements. No specific functionality has been added, but rather some small tweaks to make the core easier to understand and more streamlined.

**Added**
- Readlength set in config file
- Index for either STAR or STAR/RSEM depending on aligner chosen
- Can skip the reporting step
- Improved logging
- Renamed main script file as a separate wrapper for handling logic of starting pipeline will be built

**To test**
```
git clone git@github.com:ClinicalGenomicsGBG/qd-rnaseq-wrapper.git
cd qd-rnaseq-wrapper
git checkout optimise_one
#(optional step if no conda env built)
conda env create -f conda-env/qd-rnaseq.yaml -p conda-env/qd-rnaseq
conda activate conda-env/qd-rnaseq
python runner.py --outdir ./test_outdir --testrun
```

### CODE REVIEW

File: runner.py [hunk 1]
```diff
@@ -143,13 +149,17 @@ def main(
     finished_pipes = start_pipe_threads(pipe_commands, logger)
 
     # Move selected files to report dir
-    logger.info(f'Moving files to {config.get("general", "report_dir")}')
-    transferred_files = report_results(finished_pipes, outdir, sample_name, config)
-    for pipe, transfers in transferred_files.items():
-        logger.info(f"Moved {transfers} files generated by {pipe}")
+    if skip_report:
+        logger.info('Reporting disabled, skipping copy to report dir')
```
Comment (dodslaser) [inline runner.py]: Black should replace these single quotes with double quotes (if we're going for black codestyle)

File: runner.py [hunk 2]
```diff
@@ -143,13 +149,17 @@ def main(
...
+    else:
+        report_dir = os.path.join(config.get("general", "report_dir"), sample_name)
+        logger.info(f'Moving files to {report_dir}')
```
Comment (dodslaser) [inline runner.py]: These single quotes should also be double quotes

**Review by dodslaser — APPROVED** [review]

**Review by alvaralmstedt — APPROVED** [review]

File: runner.py (hunk 1 above)
Comment (pbiology) [reply runner.py]: Fixed in ee29a39

File: runner.py (hunk 2 above)
Comment (pbiology) [reply runner.py]: Fixed in ee29a39

**Pull request merged**